packages/hint/docs/user-guide/concepts/formatters.md

```diff
@@ -1,72 +1,92 @@
 # Formatters
 
-A `formatter` takes the results of executing the configured hints and
-transforms them to be consumed by the user. A `formatter` can output
-the results via the `console` in different formats, a `JSON` file,
-`XML`, etc.
+A formatter takes the results of executing the configured hints and
+transforms them to be consumed by the user. A formatter can output
+results to a file or the console, in various styles.
 
-You can specify one or more `formatter`s as the output. E.g.: You want
-a summary in the screen as well as a text report. You need to add the
-name inside the property `formatters`:
+To choose a formatter, install its package and add that package name to
+the `formatters` array within your .hintrc file. Packages within the
+`@hint/` namespace (like, for example, `@hint/formatter-html`) can be
+added using their short name.
 
 ```json
 {
-    "formatters": ["formatter1"]
+    "formatters": ["html"]
 }
 ```
 
-or
+Since .hintrc accepts formatters as an array of strings, you can specify
+more than one. An example use case would be wanting a summary printed to
+the console as well as a full html report.
 
 ```json
 {
     "formatters": [
-        "formatter1",
-        "formatter2"
+        "summary",
+        "html"
     ]
 }
 ```
 
-## List of official `formatter`s
+Or perhaps you wish you use a custom formatter in addition to or instead
+of the official packages. Any package matching the pattern
+`@hint/formatter-`, `webhint-formatter-`, or
+`@scope/webhint-formatter-` should be a valid candidate.
 
-The officially supported `formatter`s that can be installed via `npm` are:
+```bash
+npm i -D @myOrg/webhint-formatter-bubble-letters
+```
+
+```json
+{
+    "formatters": [
+        "html",
+        "@myorg/webhint-formatter-bubble-letters"
+    ]
+}
+```
+
+## List of official formatters
+
+The officially supported formatters that can be installed via `npm` are:
 
 * [`@hint/formatter-json`][formatter-json] does a `JSON.stringify()` of
   the results. Output is not user friendly:
 
-![Example output for the json formatter](images/json-output.png)
+  ![Example output for the json formatter](images/json-output.png)
```

Comment:
IIRC we had some problems rendering images (or maybe code?) "inside" lists in the website back in the day.
I prefer how you've done it @gesa so I'll try to test the docs locally to make sure we don't have that problem before merging.